<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Записная книжка</string>
  </property>
  <widget class="QWidget" name="MyNotes">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="main_layout">
         <item>
          <layout class="QHBoxLayout" name="header_layout">
           <item>
            <layout class="QVBoxLayout" name="input_layout">
             <item>
              <layout class="QHBoxLayout" name="name_layout">
               <item>
                <widget class="QLabel" name="name_label">
                 <property name="text">
                  <string>Имя</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="contactName"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="telephone_layout">
               <item>
                <widget class="QLabel" name="number_label">
                 <property name="text">
                  <string>Телефон</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="contactNumber"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="addContactBtn">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="contactList"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
